### PR TITLE
DevKit updates for 1.x branch

### PR DESCRIPTION
**Pull request by SonataCI**
## Changelog

```markdown
### Deprecated
- EntityManagerMockFactoryTrait
```

### CODE REVIEW

File: src/Test/EntityManagerMockFactoryTrait.php [hunk 1]
```diff
@@ -21,6 +21,11 @@
 use Doctrine\Persistence\Mapping\ClassMetadata;
 use PHPUnit\Framework\MockObject\MockObject;
 
+/**
+ * NEXT_MAJOR: Remove this class.
```
Comment (jordisala1991) [inline src/Test/EntityManagerMockFactoryTrait.php:25]: IMO We should deprecate this trait, is makes no sense to expose as a trait a code that mocks the entity manager. wdyt @VincentLanglet ?

I only saw it used on ClassificationBundle, and we can easily adapt that code.

**Review by jordisala1991 — CHANGES_REQUESTED** [review]
Waiting for @VincentLanglet

File: src/Test/EntityManagerMockFactoryTrait.php (hunk 1 above)
Comment (VincentLanglet) [reply src/Test/EntityManagerMockFactoryTrait.php:25]: Sure, it's used in https://github.com/sonata-project/SonataClassificationBundle/blob/ae001e0928c83236343319e1610512bad73f43a2/tests/Entity/TagManagerTest.php

Can you provide a PR to stop using it ?

**Review by VincentLanglet — DISMISSED** [review]

**Comment by VincentLanglet** [comment]
Changelog is needed

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**